apps/meteor/server/methods/setAvatarFromService.ts

```diff
@@ -29,7 +31,8 @@ Meteor.methods({
 		let user;
 
 		if (userId && userId !== Meteor.userId()) {
-			if (!hasPermission(Meteor.userId(), 'edit-other-user-avatar')) {
+			// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
```

Comment:
instead of having to disable this rule, you can make your code complaint. one way to achieve that is assigning `Meteor.userId()` value to a variable and checking/using that variable, so something like:
```ts
const userId = Meteor.userId();
if (!userId) {
  throw new Error('generic error');
}

// now you can use userId here, no need to disable a TS lint rule
if (!hasPermission(userId, 'edit-other-user-avatar')) {
  // throw ...
}
```